collections/_article/godotsharp-packages-net8.md

```diff
@@ -0,0 +1,68 @@
+</figure>
+
+## What does this mean for users?
+
+This means your project now needs to target .NET 8 or newer, otherwise it will be incompatible with the new `GodotSharp` packages. But you can still use libraries that target older .NET versions.
+
+Existing projects will be automatically upgraded to target `net8.0` when they are opened with Godot 4.4. If your project already targets .NET 8 or newer, nothing will change. Thanks to [Andreia Gaita](https://github.com/shana) for implementing this in [GH-82927](https://github.com/godotengine/godot/pull/82927).
+
+## What took so long?
+
+The version of .NET that a library targets becomes the minimum required version for consumers of that library. That is, if the `GodotSharp` package targets `net7.0`, your game project needs to target `net7.0` or newer. We didn't want to force our users to upgrade to a non-LTS release.
+
+When .NET 8 was released, it would've been a good time to bump the target version in Godot C# packages, since .NET 8 is an LTS release. Unfortunately, there were some breaking changes in .NET 7[^1] that prevented us from upgrading and making the required changes took us longer than we hoped.
+
+However, users could always use the latest version of .NET in their projects. Up until now, .NET 6 has been the minimum required version, but newer versions have always been supported from day one (even the pre-releases[^2]). In 4.4, the new minimum is now .NET 8, but as always, your projects can target the recent STS release .NET 9 and any newer version that is released in the future.
+
+Libraries tend to stay on older versions of .NET to support as many consumers as possible. But upgrading to the latest version of .NET brings benefits that we want to take advantage of.
+
+[^1]: .NET 7 introduced the new `scoped` keyword. This feature breaks compatibility in some low-level scenarios that affected Godot.
+
+[^2]: While pre-releases are usually supported by Godot, they may require users to enable them by setting the `DOTNET_ROLL_FORWARD_TO_PRERELEASE` environment variable to `1`. Keep in mind pre-releases are still a work-in-progress so there may be bugs.
+
+## What about older Godot releases?
+
+We encourage users to upgrade to the latest version of Godot. We work hard to prevent breaking compatibility, to ensure you can safely upgrade to the next version of Godot when it's released. Please, make sure to test the pre-releases and let us know if you find some bugs so we can fix them before the stable release.
+
+Older versions of Godot will keep targeting .NET 6, we won't change the target version in patch updates, since that would be a big breaking change. So if you still need to target .NET 6 you can stay on an older version of Godot, but we strongly encourage you to update to benefit from all the fixes and improvements included in the latest version of Godot and the .NET runtime.
```

Comment:
```suggestion
Older versions of Godot will keep targeting .NET 6; we won't change the target version in patch updates, since that would be a big breaking change. If you still need to target .NET 6, you can stay on an older version of Godot, but we strongly encourage you to update to 4.4 to benefit from all the fixes and improvements included in the latest version of Godot and the .NET runtime.
```

Reply:
"but we strongly encourage you to update" was meant to refer to both Godot and .NET, so I feel like adding "to 4.4" could give the wrong idea.

I wanted to keep this short, but if we want to be specific we'd need to mention:
- Updating Godot to 4.4
- Updating the .NET SDK to the latest version (currently 9)
- Updating the target .NET in your `.csproj` to the latest version (currently 9, but some may prefer to keep it at 8 since they have  policy to only update between LTS versions)